code/workspaces/auth-service/src/userManagement/authZeroUserManagement.js

```diff
@@ -7,19 +7,19 @@ import { getOrSetCacheAsyncWrapper } from '../cache/cache';
 
 export const authZeroManagementApi = 'https://mjbr.eu.auth0.com/api/v2';
 
-const accessTokenRequest = {
+const accessTokenRequest = () => ({
```

Comment:
Why did you make this a function?

Reply:
Good question. I needed to be a function to defer the import for mocking out in the tests. This was an issue on for running the test in the local environment. Whereby if the environmental variable was set the test would fail. This has now been switched to have a fixed value when testing.